ui/src/pages/Archives/index.tsx

```diff
@@ -1,6 +1,83 @@
+import { Box, Grid, Typography } from '@material-ui/core'
+import React, { useEffect, useState } from 'react'
+
+import { Archive } from 'api/archives.type'
+import ArchiveOutlinedIcon from '@material-ui/icons/ArchiveOutlined'
+import ConfirmDialog from 'components/ConfirmDialog'
 import ContentContainer from 'components/ContentContainer'
-import React from 'react'
+import ExperimentPaper from 'components/ExperimentPaper'
+import Loading from 'components/Loading'
+import api from 'api'
 
 export default function Archives() {
-  return <ContentContainer>Archives</ContentContainer>
+  const [loading, setLoading] = useState(false)
+  const [archives, setArchives] = useState<Archive[] | null>(null)
+  const [selected, setSelected] = useState({
+    namespace: '',
+    name: '',
+    kind: '',
+    title: '',
+    description: '',
+    action: 'recover',
+  })
+  const [dialogOpen, setDialogOpen] = useState(false)
+
+  const fetchArchives = () => {
+    setLoading(true)
+
+    api.archives
+      .archives()
+      .then(({ data }) => setArchives(data))
+      .catch(console.log)
+      .finally(() => {
+        setLoading(false)
+      })
+  }
+
+  useEffect(fetchArchives, [])
+
+  const handleArchive = (action: string) => () => {
+    switch (action) {
+      case 'recover':
+        break
+
+      default:
+        break
+    }
+  }
+
+  return (
+    <ContentContainer>
+      <Grid container spacing={3}>
+        {archives &&
+          archives.length > 0 &&
+          archives.map((a) => (
+            <Grid key={a.Name} item xs={12}>
+              <ExperimentPaper experiment={a} isArchive handleSelect={setSelected} handleDialogOpen={setDialogOpen} />
```

Comment:
Do we need to specify `isArchive={true}` here? If not specified, I thought it will use the default value false

Reply:
No need, if the boolean prop not explicitly specified, It will be considered as true.